Kernel/FileSystem/FATFS/Inode.cpp

```diff
@@ -57,7 +57,7 @@ ErrorOr<Vector<BlockBasedFileSystem::BlockIndex>> FATInode::compute_block_list()
 
         auto first_block_and_length = fs().first_block_of_cluster(cluster);
         for (u8 i = 0; i < first_block_and_length.number_of_sectors; i++)
-            block_list.append(BlockBasedFileSystem::BlockIndex { first_block_and_length.start_block.value() + i });
+            TRY(block_list.try_append(BlockBasedFileSystem::BlockIndex { first_block_and_length.start_block.value() + i }));
```

Comment:
Using `Better` as a verb is stretching it, but I'll allow it
(`Kernel/FATFS: Better error propagation in FATInode`)